common/src/View/MapDocument.h

```diff
@@ -31,6 +31,8 @@
 #include <vecmath/bbox.h>
 #include <vecmath/util.h>
 
+#include <QString>
```

Comment:
I would much prefer it if we could try and keep Qt types out of `MapDocument`.

Reply:
I don't see any reason to avoid it, MapDocument is in the view module and QString is the correct type

Reply:
I'd like to split "View" into "UI" and "Whatever" at some point, and only "UI" should depend on Qt. At the same time, I don't see `MapDocument` as a part of UI.

Since we have done the Qt port, I am very mindful of not importing Qt into all parts of TB. If at all possible, I want it in the UI only.